docs/source/features/version_control/tag.rst

```diff
@@ -0,0 +1,87 @@
+..
+ Copyright 2021 Graviti. Licensed under MIT License.
+ 
+#####
+ Tag
+#####
+
+Graviti supports tagging specific commits in a dataset's history as being important.
+Or use this feature to mark release revisions (v1.0, v2.0 and so on).
```

Comment:
Should we use `or` here? Isn't `mark release revisions` tagging important commits?